src/validation/custom.validation.ts

```diff
@@ -630,6 +630,29 @@ export const validateEmail = (email: string, maxLength: number) => {
   return true;
 };
 
+export const checkPropertyNameOrNumberIfRadioButtonSelected = (selected: boolean, propertyNameOrNumber: string, req) => {
+  if (selected) {
+    return checkPresenceOfPropertyNameOrNumber(propertyNameOrNumber, req);
+  }
+
+  return true;
+};
+
+export const checkPresenceOfPropertyNameOrNumber = (propertyNameOrNumber: string, req) => {
+  if (propertyNameOrNumber === undefined || propertyNameOrNumber === "") {
```

Comment:
I copied the approach taken for the 'email' field here, but if we do ever implement this, we may wish to `trim` the value too before testing it.